include/swift/AST/Identifier.h

```diff
@@ -206,15 +208,45 @@ namespace swift {
 
 /// Wrapper that may either be an Identifier or a special name
 /// (e.g. for subscripts)
+///
+/// The raw values of this enum are used in serialization by both
```

Comment:
I'm still using them for serialisation in ClangImporter but you said that they don't need to be stable for that, correct?